compute/instances/custom-machine-type/custom_machine_type_test.go

```diff
@@ -166,4 +166,92 @@ func TestComputeCreateInstanceWithCustomMachineTypeSnippets(t *testing.T) {
 	if err != nil {
 		t.Errorf("deleteInstance got err: %v", err)
 	}
+
+	buf.Reset()
+
+	want = "Instance updated"
+
+	if err := createInstanceWithCustomMachineType(buf, tc.ProjectID, zone, instanceName, customMT); err != nil {
+		t.Errorf("createInstanceWithCustomMachineType got err: %v", err)
```

Comment:
for these test consider if the test is still valid if a precondition fails. For example if the instance is never created in this step you are unable to modify or get the instance so this should fail the test with a t.Fatalf.